Source/TutorialController.swift

```diff
@@ -40,20 +37,27 @@ let MinimumMarginLateralSpace: CGFloat = 20.0
 
   // MARK: UIAppearance
 
-  @objc public static func setFont(font: UIFont) {
+  @objc public static func setTitleFont(font: UIFont) {
     UILabel.appearance().font = font
   }
 
-  @objc public static func setTextColor(color: UIColor) {
+  @objc public static func setTextFont(font: UIFont) {
+    UITextView.appearance().font = font
+  }
+
+  @objc public static func setTitleColor(color: UIColor) {
     UILabel.appearance().textColor = color
   }
 
+  @objc public static func setTextColor(color: UIColor) {
+    UITextView.appearance().textColor = color
+  }
+
   // MARK: Device orientation
 
   func didRotate() {
-    layoutSubviews()
+    //layoutSubviews()
```

Comment:
Method is not used now, kept it just for a case we'll need to do something when orientation is changed. But can remove it.